packages/server/src/seed/seed.controller.ts

```diff
@@ -56,6 +58,8 @@ export class SeedController {
     await this.seedService.deleteAll(CourseSectionMappingModel);
     await this.seedService.deleteAll(CourseModel);
     await this.seedService.deleteAll(SemesterModel);
+    await this.seedService.deleteAll(OrganizationCourseModel);
+    await this.seedService.deleteAll(OrganizationUserModel);
```

Comment:
Shouldn't the OrganizationUserModel.userId be a forgein key? 
Right now it is possible to remove the user from the user table but they are still an organization user.

Same thing with OrganizationCourseModel.

Use `@JoinColumn` in the entities. See `user-course.entity.ts` 

After adding foreign keys, these delete staments need to be in the correct order.

Reply:
I will delete move deletion to the top, that makes sense to me.